trackthenews/core.py

```diff
@@ -425,24 +413,20 @@ def setup_matchlist():
         with open(path, "w") as f:
             f.write("")
         print(
-            "A new matchlist has been generated at {path}. You can add case insensitive entries to match, one per line.".format(
-                **locals()
-            )
+            "A new matchlist has been generated at {path}."
+            " You can add case insensitive entries to match, one per line.".format(**locals())
```

Comment:
Wow I did not realize this sort of multi-line formatting would work.

Future: This change didn't introduce this line, but probably we should eventually use an f-string here and in the other locations that use `**locals()` for string formatting data